web/api/web_api_v1.c

```diff
@@ -1564,6 +1787,11 @@ static struct web_api_command api_commands_v1[] = {
         { "alarm_count",     0, WEB_CLIENT_ACL_DASHBOARD_ACLK_WEBRTC, web_client_api_request_v1_alarm_count,     0              },
         { "allmetrics",      0, WEB_CLIENT_ACL_DASHBOARD_ACLK_WEBRTC, web_client_api_request_v1_allmetrics,      0              },
 
+#if defined(ENABLE_LOGSMANAGEMENT)
```

Comment:
Please,

Address this warning:

```sh
web/api/web_api_v1.c:1791:9: warning: missing initializer for field 'allow_subpaths' of 'struct web_api_command' [-Wmissing-field-initializers]
 1791 |         { "logsmanagement_sources", 0, WEB_CLIENT_ACL_DASHBOARD_ACLK_WEBRTC, web_client_api_request_v1_logsmanagement_sources   },
```

Reply:
There is another for next line:

```sh
web/api/web_api_v1.c:1792:9: warning: missing initializer for field 'allow_subpaths' of 'struct web_api_command' [-Wmissing-field-initializers]
 1792 |         { "logsmanagement",         0, WEB_CLIENT_ACL_DASHBOARD_ACLK_WEBRTC, web_client_api_request_v1_logsmanagement           },
      |         ^
web/api/web_api.h:34:18: note: 'allow_subpaths' declared here
   34 |     unsigned int allow_subpaths;
      |                  ^~~~~~~~~~~~~~
```

It is possible this was missed during rebase.

Reply:
Somehow I missed those, thanks. Should be fixed now.